src/net/sourceforge/kolmafia/persistence/DebugDatabase.java

```diff
@@ -92,7 +84,7 @@ private static String readWikiData(String url) {
           byte[] bytes = ByteBufferUtilities.read(istream);
           return StringUtilities.getEncodedString(bytes, "UTF-8");
         }
-
+        // JAA - Should this be && and not || ?
```

Comment:
I'd like someone else's opinion.  Should this be (responseCode >= 301 && responseCode < 308) ?

Reply:
I mean I suppose you could write `floor(responseCode/100) == 3`